Lux/lux_files/kaleido.json

```diff
@@ -95,7 +95,13 @@
             "min": -10,
             "max": 10,
             "step": 1,
-            "default_value": 0
+            "default_value": 0,
+            "audio_reactive": {
```

Comment:
audio_reactive doesn't appear anywhere in slider functor or in scene_io
instead of hard coding this in, put the audio function into the scene file and include it in the slider's harnesses
e.g. `            "value": { "functions": [ "audio_high_band" ] }`

Reply:
Look at main branch where I do this with the `tweaker` function
``` 
       {
            "name": "segment_slider",
            "type": "slider_float",
            "label": "Segments",
            "description": "Number of segments in the kaleidoscope image",
            "min": 1,
            "max": 36,
            "step": 1,
            "default_value": 6,
            "value": { "functions": [ "segment_tweaker" ] }
        },    
         ```